recipes/cctk/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "cctk" %}
+{% set version = "0.2.18" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 83881c5ce07edcb5b45b9e4d7771992974c1cf622d8ef272d4f35e9f28889037
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - basis_set_exchange
+    - importlib_resources
+    - networkx
+    - numpy
+    - pip
+    - pyahocorasick
+    - python >=3.7
+    - pyyaml
+    - scipy
```

You don't need all the runtime dependencies at build time, unless the package imports itself at build time, then you should report that upstream. BTW, the metadata says `python_requires='>=3.6'`, not 3.7.

```suggestion
    - python >=3.6
    - pip
```